libraries/lib-sentry-reporting/SentryReport.cpp

```diff
@@ -38,6 +39,19 @@ namespace sentry
 {
 namespace
 {
+
+//! Remove paths from the log    
+std::string AnonymizePaths(const std::string& input)
+{
+   // Finding the path boundary in the arbitrary text is a hard task.
+   // We assume that spaces cannot be a part of the path. 
+   // In the worst case - we will get <path> <path> 
+   // Also, we assume, that
+   static const std::regex re(
+      R"(\b(?:(?:[a-zA-Z]:)?[\\/]?)?(?:[^<>:"/|\\/?\s*]+[\\/]+)*(?:[^<>:"/|\\/?*\s]+\.\w+)?)");
```

Comment:
Raw string literals are good.

A concatenation of raw string literals, with comments explaining this big regex, would be even more helpful to a reader.